biztalk/technical-guides/checklist-measuring-performance-on-hyper-v.md

```diff
@@ -23,14 +23,14 @@ While most of the principles of analyzing performance of a guest operating syste
hout cache. The use of logical disk versus physical disk performance monitor counters is recommended because Windows applications and services utilize logical drives represented as drive letters wherein the physical disk (LUN) presented to the operating system can be comprised of multiple physical disk drives in a disk array. Use the following rule of thumb when measuring disk latency on the Hyper-V host operating system using the \Logical Disk(\*)\Avg. Disk sec/Read or \Logical Disk(\*)\Avg. Disk sec/Write performance monitor counters:<br /><br /> This is based on the typical seek time of a single 7200 RPM disk drive without cache. The use of logical disk versus physical disk performance monitor counters is recommended because Windows applications and services utilize logical drives represented as drive letters wherein the physical disk (LUN) presented to the operating system can be comprised of multiple physical disk drives in a disk array. Use the following rule of thumb when measuring disk latency on the Hyper-V host operating system using the \Logical Disk(\*)\Avg. Disk sec/Read or \Logical Disk(\*)\Avg. Disk sec/Write performance monitor counters:<br /><br /> -   1ms to 15ms  = Healthy<br />-   15ms to 25ms = Warning or Monitor<br />-   26ms or greater = Critical, performance will be adversely affected \*\*Note:*\*  Physical disks installed on a non-virtualized environment offer better performance than disks accessed through a Hyper-V host operating system. If disk performance is absolutely critical to the overall performance of your application, consider hosting disks on physical hardware only. **Note:**  When evaluating disk I/O performance, ensure that you configure antivirus software to exclude scanning of any disk partitions that are being evaluated. Antivirus scanning introduces overhead that can negatively impact performance and skew test results. <br /><br /> **Measure disk latency on guest operating systems** – Response times of the disks used by the guest operating systems can be measured using the same performance monitor counters used to measure response times of the disks used by the Hyper-V host operating system. | For more information about disk performance analysis, see the following resources:<br /><br /> -   “Performance Overhead of Running SQL Server in Hyper-V” section of “Running SQL Server 2008 in a Hyper-V Environment – Best Practices and Performance Considerations” whitepaper [here](https://go.microsoft.com/fwlink/?LinkId=144622).<br />-   Ruling Out Disk-Bound Problems [here](https://go.microsoft.com/fwlink/?LinkId=120947).<br />-   SQL Server Predeployment I/O Best Practices<br />-   “I/O Bottlenecks” section of the “Troubleshooting Performance Problems in SQL Server 2008” whitepaper available [here](https://go.microsoft.com/fwlink/?LinkID=202135).<br />-   How to Identify a Disk Performance Bottleneck Using the Microsoft Server Performance Advisor (SPA) Tool [here](https://go.microsoft.com/fwlink/?LinkID=98096). |
```

Comment:
```suggestion
| **Measure disk latency on a Hyper-V host operating system** – The best initial indicator of disk performance on a Hyper-V host operating system is obtained by using the “\Logical Disk(*)\Avg. Disk sec/Read” and “\Logical Disk(\*)\Avg. Disk sec/Write” performance monitor counters. These performance monitor counters measure the amount of time that read and write operations take to respond to the operating system. As a general rule of thumb, average response times greater than 15ms are considered sub-optimal. As a general rule of thumb, average response times greater than 15ms are considered sub-optimal. This is based on the typical seek time of a single 7200 RPM disk drive without cache. The use of logical disk versus physical disk performance monitor counters is recommended because Windows applications and services utilize logical drives represented as drive letters wherein the physical disk (LUN) presented to the operating system can be comprised of multiple physical disk drives in a disk array. Use the following rule of thumb when measuring disk latency on the Hyper-V host operating system using the \Logical Disk(\*)\Avg. Disk sec/Read or \Logical Disk(\*)\Avg. Disk sec/Write performance monitor counters:<br /><br /> This is based on the typical seek time of a single 7200 RPM disk drive without cache. The use of logical disk versus physical disk performance monitor counters is recommended because Windows applications and services utilize logical drives represented as drive letters wherein the physical disk (LUN) presented to the operating system can be comprised of multiple physical disk drives in a disk array. Use the following rule of thumb when measuring disk latency on the Hyper-V host operating system using the \Logical Disk(\*)\Avg. Disk sec/Read or \Logical Disk(\*)\Avg. Disk sec/Write performance monitor counters:<br /><br /> -   1ms to 15ms  = Healthy<br />-   15ms to 25ms = Warning or Monitor<br />-   26ms or greater = Critical, performance will be adversely affected \*\*Note:*\*  Physical disks installed on a non-virtualized environment offer better performance than disks accessed through a Hyper-V host operating system. If disk performance is absolutely critical to the overall performance of your application, consider hosting disks on physical hardware only. **Note:** When evaluating disk I/O performance, ensure that you configure antivirus software to exclude scanning of any disk partitions that are being evaluated. Antivirus scanning introduces overhead that can negatively impact performance and skew test results. <br /><br /> **Measure disk latency on guest operating systems** – Response times of the disks used by the guest operating systems can be measured using the same performance monitor counters used to measure response times of the disks used by the Hyper-V host operating system. | For more information about disk performance analysis, see the following resources:<br /><br /> - “Performance Overhead of Running SQL Server in Hyper-V” section in the [“Running SQL Server 2008 in a Hyper-V Environment – Best Practices and Performance Considerations” whitepaper](https://go.microsoft.com/fwlink/?LinkId=144622) <br />- [Ruling Out Disk-Bound Problems](https://go.microsoft.com/fwlink/?LinkId=120947) <br />- [SQL Server Predeployment I/O Best Practices](https://seniordba.wordpress.com/2016/04/19/sql-server-io-best-practice/) <br />- “I/O Bottlenecks” section in the [“Troubleshooting Performance Problems in SQL Server 2008” whitepaper](https://go.microsoft.com/fwlink/?LinkID=202135) <br />- [How to Identify a Disk Performance Bottleneck Using the Microsoft Server Performance Advisor (SPA) Tool](https://go.microsoft.com/fwlink/?LinkID=98096) |
```